bcda/models/alr.go

```diff
@@ -23,3 +24,17 @@ type AlrMetaData struct {
 	ACO       string
 	Timestamp time.Time
 }
+
+// Data Structure for Jobs
+type AlrJobs struct {
```

Comment:
We should have a new table for this right?

Reply:
Yes. Should the migration be included in this ticket?

Reply:
Yeah, let's put it in this ticket